src/form.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react'
+
+const styles = {
+  form: {
+    width: '25rem',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    border: '1px solid black',
+    padding: '2rem',
+  },
+  title: {
+    fontSize: '1.5rem',
+    textAlign: 'center',
+    padding: '1rem'
+  }
+}
+
+export default class Form extends Component {
+  constructor(props) {
+    super(props)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  handleSubmit(e) {
+    e.preventDefault()
+    const flashcard = {
+      question: document.querySelector('#question').value,
```

Comment:
Instead of querying the document, use a [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData) object to [`get`](https://developer.mozilla.org/en-US/docs/Web/API/FormData/get) the values. When building React apps, we try not to touch the DOM directly.